### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by NextNebula**
Updated the Dutch translation with five new translations, fixed some spelling mistakes and made the translation for 'folder' consistent to the word 'map' that is commonly used in Dutch.

### CODE REVIEW

File: lang/nl.php [hunk 1]
```diff
@@ -16,43 +17,43 @@
...
+    "Missing a temporary folder." => "Een tijdelijke map ontbreekt.",
     "Failed to write file." => "Poging tot schrijven van bestand is mislukt.",
     "Denied file extension." => "De extensie van dit bestand is niet toegestaan.",
     "Unknown image format/encoding." => "Onbekende afbeeldingsformaats/-codering.",
     "The image is too big and/or cannot be resized." => "De afbeelding is te groot en/of de grootte kan niet aangepast worden.",
     "Cannot create {dir} folder." => "Kan de map {dir} niet aanmaken.",
-    "Cannot rename the folder." => "De folder kan niet hernoemd worden.",
-    "Cannot write to upload folder." => "Kan niet naar de upload folder schrijven.",
+    "Cannot rename the folder." => "De map kan niet hernoemd worden.",
+    "Cannot write to upload folder." => "Kan niet naar de upload map schrijven.",
```
Comment (marcovtwout) [inline lang/nl.php]: Spatie fout ;)

File: lang/nl.php [hunk 2]
```diff
@@ -16,43 +17,43 @@
...
     "Cannot read .htaccess" => "Kan .htaccess niet lezen.",
-    "Incorrect .htaccess file. Cannot rewrite it!" => "Incorrect .htaccess bestand. Bestand kan niet herschreven worden!",
-    "Cannot read upload folder." => "Upload folder kan niet uitgelezen worden.",
-    "Cannot access or create thumbnails folder." => "Het is niet mogelijk om een miniatuurweergaven folder aan te maken of te benaderen.",
-    "Cannot access or write to upload folder." => "Het is niet mogelijk om in de upload folder te schrijven of deze te benaderen.",
-    "Please enter new folder name." => "Vul a.u.b. een nieuwe foldernaam in.",
-    "Unallowable characters in folder name." => "Er zijn niet toegestane karakters gebruikt in de foldernaam.",
-    "Folder name shouldn't begins with '.'" => "Een foldernaam mag niet met '.' beginnen.",
+    "Incorrect .htaccess file. Cannot rewrite it!" => "Verkeerd .htaccess bestand. Bestand kan niet herschreven worden!",
+    "Cannot read upload folder." => "Upload map kan niet uitgelezen worden.",
```
Comment (marcovtwout) [inline lang/nl.php]: Another one here. Could you check the whole file while you're at it? :)
http://www.spatiegebruik.nl/regels.html

**Comment by fourroses666** [comment]
is this translated to?

c3f829b0-f782-11e3-85b5-51df1e67a3f5

**Comment by NextNebula** [comment]
The words that are still English in your screengrap are translated in this language file. Also tested it.

image

**Comment by fourroses666** [comment]
thanks for the confirm.

**Pull request merged**